languageserver/src/connection.ts

```diff
@@ -51,7 +51,7 @@ export function initConnection(connection: Connection) {
     const options = params.initializationOptions as InitializationOptions;
 
     if (options.sessionToken) {
-      client = getClient(options.sessionToken, options.userAgent);
+      client = getClient(options.sessionToken, options.userAgent, options.githubApiUrl);
```

Comment:
matching the comment in the other PR:

```suggestion
      client = getClient(options.sessionToken, options.userAgent, options.gitHubApiUrl);
```